project_euler/problem_18/sol1.py

```diff
@@ -0,0 +1,64 @@
+"""
+By starting at the top of the triangle below and moving to adjacent numbers on the row below, the maximum total from top to bottom is 23.
```

Comment:
Reading this comment on GitHub requires left/right scrolling which reduces readability.  Please wrap all lines at 88 characters max.